astropy/utils/iers/tests/test_iers.py

```diff
@@ -304,3 +304,24 @@ def test_simple(self):
             # Now the time range should be different.
             assert dat['MJD'][0] == 57359.0 * u.d
             assert dat['MJD'][-1] == (57539.0 + 60) * u.d
+
+
+@pytest.mark.remote_data
+def test_IERS_B_parameters_loading_into_IERS_Auto():
+    A = iers.IERS_Auto.open()
+    B = iers.IERS_B.open()
+
+    ok_A = A["MJD"] <= B["MJD"][-1]
+    assert not np.all(ok_A), "IERS B covers all of IERS A: should not happen"
+
+    i_B = np.searchsorted(B["MJD"], A["MJD"][ok_A])
```

Comment:
This seems subject to off-by-one index errors for times where `A["MJD"]` is just slightly more (float error) than the corresponding `B["MJD"]`.  Maybe it doesn't matter?  I think in situations like this I have used nearest neighbor interpolation on an index array.  However, I don't know if that can be easily done without scipy interpolate.

Reply:
The MJD are int, so I think it is OK.

Reply:
(well, they are float but represent `int` values)

Reply:
Ah, then fine!